test/tests/functional/pbs_test_run_count.py

```diff
@@ -154,14 +154,14 @@ def test_run_count_subjob_in_x(self):
 
         a = {ATTR_J: '1-6'}
         j = Job(TEST_USER, a)
-        j.set_sleep_time(10)
+        j.set_sleep_time(20)
         jid = self.server.submit(j)
-        time.sleep(9)
+        time.sleep(15)
         self.server.expect(JOB, {ATTR_state: "R"},
                            id=j.create_subjob_id(jid, 2))
         # Create an execjob_begin hook that rejects the job
         self.create_reject_begin_hook()
-        time.sleep(8)
+        time.sleep(15)
```

Comment:
Make both of these sleeps offsets in the expect.  Also consider adding a logger message explaining what you are doing.